.github/workflows/conda-package.yml

```
@@ -177,8 +176,8 @@ jobs:
       - name: Run tests
         if: env.RERUN_TESTS_ON_FAILURE != 'true'
         run: |
-          python -m pytest -q -ra --disable-warnings -vv .
-        working-directory: ${{ env.tests-path }}
+          python -m pytest -ra --pyargs ${{ env.MODULE_NAME }}.tests
```

It seems we have duplicating env variables `PACKAGE_NAME` and `MODULE_NAME`. Would it make sense to use only one of it?
```suggestion
          python -m pytest -ra --pyargs ${{ env.PACKAGE_NAME}}.tests
```

---

Done
Thank you!